### PR TITLE
Updating Microsoft Core and Microsoft ADFS Parsing

### PR DESCRIPTION
**Pull request by nkanon**
## Status
- [x] In Progress
- [ ] Ready
- [ ] In Hold - (Reason for hold)

## Description
This PR is created in order of changing the parsing rules of ADFS and the Microsoft Core packs.

## Minimum version of Cortex XSOAR
- [ ] 6.0.0
- [ ] 6.1.0
- [ ] 6.2.0
- [ ] 6.5.0
- [x] 8.2.0

## Does it break backward compatibility?
   - [ ] Yes
       - Further details:
   - [x] No

### CODE REVIEW

File: Packs/MicrosoftADFS/README.md [hunk 1]
```diff
@@ -27,36 +26,20 @@ To view logs only from the Windows AD FS, apply the following filter to the data
...
-    event_id: 510, 1200, 1201, 1202, 1203, 1204, 1205, 1206, 1207
-    processors:
-      - add_fields:
-          fields:
-            vendor: microsoft
-            product: windows
-```
-​
-**Note**: The above configuration uses the default location of the logs. 
+When configuring the Windows Event Collector (WEC), please use the following settings in the **Configurations** -> **Broker VMs** -> **WEC** -> **Collection Configuration** section:
```
Comment (ShirleyDenkberg) [inline Packs/MicrosoftADFS/README.md:40]:
```suggestion
When configuring the Windows Event Collector (WEC), use the following settings in the **Configurations** > **Broker VMs** > **WEC** > **Collection Configuration** section:
```

File: Packs/MicrosoftADFS/ReleaseNotes/1_0_14.md [hunk 2]
```diff
@@ -0,0 +1,13 @@
+
+#### Parsing Rules
+
+##### Microsoft ADFS Collection Parsing Rule
+
+- Updated the Parsing Rule to point to a new dataset "microsoft_adfs_raw".
```
Comment (ShirleyDenkberg) [inline Packs/MicrosoftADFS/ReleaseNotes/1_0_14.md:6]:
```suggestion
Updated the Parsing Rule to point to a new dataset "microsoft_adfs_raw".
```

File: Packs/MicrosoftADFS/ReleaseNotes/1_0_14.md [hunk 3]
```diff
@@ -0,0 +1,13 @@
...
+
+
+#### Modeling Rules
+
+##### Microsoft ADFS Collection
+
+- Updated the Modeling Rule with the correct dataset name.
```
Comment (ShirleyDenkberg) [inline Packs/MicrosoftADFS/ReleaseNotes/1_0_14.md:13]:
```suggestion
Updated the Modeling Rule with the correct dataset name.
```

File: Packs/MicrosoftWindowsEvents/ReleaseNotes/1_0_4.md [hunk 4]
```diff
@@ -0,0 +1,6 @@
+
+#### Parsing Rules
+
+##### Microsoft Windows Events
+
+- Updated the filter for the Parsing Rule to exclude "AD FS" logs.
```
Comment (ShirleyDenkberg) [inline Packs/MicrosoftWindowsEvents/ReleaseNotes/1_0_4.md:6]:
```suggestion
Updated the filter for the Parsing Rule to exclude "AD FS" logs.
```

**Comment by ShirleyDenkberg** [comment]
@nkanon Doc review completed.

**Review by Codytmpr — APPROVED** [review]

**Pull request closed without merging**